packages/utils/src/lib/execute-process.ts

```diff
@@ -85,20 +85,20 @@ export type ProcessConfig = {
 };
 
 /**
- * Process observer object. Contains the next, error and complete function.
+ * Process observer object. Contains the onStdout, error and complete function.
  * @category Types
  * @public
- * @property {function} next - The next function of the observer (optional).
+ * @property {function} onStdout - The onStdout function of the observer (optional).
  * @property {function} error - The error function of the observer (optional).
  * @property {function} complete - The complete function of the observer (optional).
  *
  * @example
  * const observer = {
- *  next: (stdout) => console.log(stdout)
+ *  onStdout: (stdout) => console.log(stdout)
  *  }
  */
 export type ProcessObserver = {
-  next?: (stdout: string) => void;
+  onStdout?: (stdout: string) => void;
   error?: (error: ProcessError) => void;
   complete?: () => void;
 };
```

Comment:
I think all these event handlers should be prefixed by `on`, or none of them should.